src/composables/useFlawModel.ts

```diff
@@ -47,7 +47,14 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), onSaveSuccess:
     if (!validatedFlaw.success) {
       return;
     }
-    postFlaw(validatedFlaw.data)
+    // Remove any empty fields before request
+    const flawForPost: any = Object.assign({}, validatedFlaw.data);
+    Object.entries(flawForPost).forEach(([key, value])=>{
+      if (value === '') {
+        delete flawForPost[key];
+      }
+    });
```

Comment:
This is fine because it's iterating over Object.entries instead of flawForPost, but I might weakly, stylistically prefer this instead of `delete`
```suggestion
const flawForPost: any = Object.fromEntries(Object.entries(validatedFlaw.data).filter(([key, value]) => value !== ''))
```
(don't commit from this suggestion directly)

Use your preference, just my thought

Reply:
That is much deliciouser. 

TBH, this was 🍳🧠 code. Even with Object.{assign,entries}, I think for non-primitives, references rather than values are copied so `delete` is potentially hazardous (though less in this case since it's mutating validatedFlaw, which actually could have entangled references with the `flaw`)